src/Microsoft.Health.Fhir.Shared.Web/Startup.cs

```diff
@@ -50,6 +50,11 @@ public virtual void ConfigureServices(IServiceCollection services)
             {
                 fhirServerBuilder.AddSqlServer(config =>
                 {
+                    if (int.TryParse(Configuration["SqlServer:SqlCommand:CommandTimeout"], out int commandTimeout) && commandTimeout >= 0)
+                    {
+                        config.CommandTimeout = TimeSpan.FromSeconds(commandTimeout);
+                    }
+
```

Comment:
Is this needed? Likely its the config that can't automatically be parsed as a timespan, i.e. `"CommandTimeout": "00:00:30"`

Reply:
I added it as a TimeSpan to the shared components after our initial review of the work. If you want I can change the shared components to an int type and test that it works without the additional setting of the CommandTimeout value.